guide/english/css/units/index.md

```diff
@@ -91,7 +91,7 @@ Relative units change relative to the element's current font-size or other setti
 Absolute units  will be the same regardless of screen size or other settings. Some absolute units are 
 - ```px```
   - pixel
-  - pixel counts are relative to the quality of the viewing device's screen
+  - [pixel do not necessarily map to "physical" device pixels](https://webplatform.github.io/docs/tutorials/understanding-css-units/#On-CSS-pixels,-physical-units-and-scalability) (1px = 1/96 of an inch).
```

Comment:
`do` -> `doesn't`

Reply:
Also as a suggestion, please add more detail on the difference of pixel mapping from physical device pixels, simply adding an external link reduces readability towards the article.
Instead it is suitable to go through this article and then adding your knowledge in your own words (to prevent plagiarism) to this article.